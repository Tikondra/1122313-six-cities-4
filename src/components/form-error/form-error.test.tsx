import * as React from "react";
import * as renderer from "react-test-renderer";
import FormError from "./form-error";

it(`should render FormError`, function () {
  const tree = renderer
    .create(
        <FormError/>,
        {createNodeMock: () => {
          return document.createElement(`div`);
        }}
    ).toJSON();

  expect(tree).toMatchSnapshot();
});
