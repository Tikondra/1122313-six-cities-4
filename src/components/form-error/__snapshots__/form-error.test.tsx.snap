// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render FormError 1`] = `
<div>
  <p
    style={
      Object {
        "color": "red",
        "position": "absolute",
      }
    }
  >
    Отзыв не отправлен. Попробуйте позднее.
  </p>
</div>
`;
